docs/guides/sdk/tracing/anthropic.mdx

```diff
@@ -5,7 +5,7 @@ description: How to trace Anthropic calls with Latitude
 
 ## Installation
 
-import Installation from '/snippets/installation.mdx'
+import Installation from '/snippets/sdks/typescript/installation.mdx'
```

Comment:
I would keep installation.mdx and there explain that we have Typescript and Python SDKs